lab_7_llm/assets/main.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
+
+let inferSampleFromForm = async (premise, hypothesis, result) => {
+    await fetch('/infer', {
+        method: 'POST',
+        headers: {
+            'Content-type': 'application/json'
+        },
+        body: JSON.stringify({
+            question: premise.value,
+            hypothesis: hypothesis.value
+        })
+    }).then(res => {return res.json()})
```

Comment:
`async \ await` function was invented as the development of promises and getting rid of `then` and promise hell

Reply:
done